src/graphql/mutation/createDeleteMutationField.js

```diff
@@ -57,15 +58,25 @@ const createPayloadType = table =>
     description: `Contains the ${table.getMarkdownTypeName()} node deleted by the mutation.`,
     interfaces: [getPayloadInterface(table.schema)],
     fields: {
-      [table.getFieldName()]: {
```

Comment:
I really don’t want to introduce breaking changes wherever possible, could we leave this field but also have the `deleted*id` field? This field could also be helpful for non-Relay users.